NEWS.md

```diff
@@ -6,6 +6,9 @@
   `pyproject.toml` file, `reticulate` will attempt to find and use the virtual
   environment managed by Poetry for that project. (#1031)
   
+- `reticulate::install_miniconda()` default python version for the default 
```

Comment:
I'd suggest something like "The default version of Python used for the `r-reticulate` Miniconda environment installed via `reticulate::install_miniconda()` has been changed from 3.6 to 3.8."